src/component/tooltip/TooltipHTMLContent.ts

```diff
@@ -307,7 +308,9 @@ class TooltipHTMLContent {
                     : isFunction(appendTo) && appendTo(api.getDom())
         );
 
-        makeStyleCoord(this._styleCoord, zr, container, api.getWidth() / 2, api.getHeight() / 2);
+        this._clearStyleCoord = makeStyleCoord(
+            this._styleCoord, zr, container, api.getWidth() / 2, api.getHeight() / 2
+        );
```

Comment:
`makeStyleCoord` is called twice in this file, and each call return a `clean` function. 
But only the last one is called when `dispose`. 
That still works because the two call of `makeStyleCoord` use the same input `zr` and `container`, but that dependency is not explicit in the API design a little bit.

It's OK to do it like that, but currently I think a better way might be:

Do not change `makeStyleCoord`.
And a new API: `transformLocalCoordClear` in `zrender/src/core/dom.ts`
And in `TooltipHTMLContainer.ts`:
```ts
dispose() {
    const zr = this._zr;
    transformLocalCoordClear(zr && zr.painter && zr.painter.getViewportRoot(), this._container);
    // ...
}
```

Reply:
This way looks much clear!